### PR TITLE
Hotfix/mindshare font

### PR DESCRIPTION
**Pull request by jomarmontuya**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "version": "1.0.1",
   "private": true,
   "scripts": {
-    "dev": "NODE_OPTIONS='--inspect' next dev",
+    "dev": "cross-env NODE_OPTIONS='--inspect' next dev",
```
Comment (ardeay) [inline package.json]: What does this change do @jomarmontuya

**Review by ardeay — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (jomarmontuya) [reply package.json]: Hi @ardeay, I recently change to using windows as my development environment and I run into issues where I cannot run the website locally, the reason being is the node options flag don't support windows machine, adding cross-env per nextjs docs will allow to run the node options on any machine,

image

Comment (ardeay) [reply package.json]: Please remove and setup WSL

Comment (jomarmontuya) [reply package.json]: @ardeay removed please review, thank you!

**Pull request merged**